<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrlDetail</class>
 <widget class="QDialog" name="CrlDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>381</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="TextLabel1">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>Details of the Revocation list</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="image">
       <property name="minimumSize">
        <size>
         <width>95</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>95</width>
         <height>40</height>
        </size>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabview">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="unnamed">
      <attribute name="title">
       <string>&amp;Status</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>8</number>
       </property>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>8</number>
          </property>
          <item row="2" column="1">
           <layout class="QHBoxLayout">
            <item>
             <widget class="ClickLabel" name="signCheck"/>
            </item>
            <item>
             <widget class="QLabel" name="TextLabel1_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Version</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="CopyLabel" name="version">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="TextLabel6">
            <property name="text">
             <string>Signature</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="TextLabel4">
            <property name="text">
             <string>Signed by</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="TextLabel2">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="descr">
            <property name="toolTip">
             <string>The internal name of the CRL in the database</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ClickLabel" name="issuerIntName"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>issuing dates</string>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>8</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="TextLabel2_2">
            <property name="text">
             <string>Next Update</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="TextLabel1_2">
            <property name="text">
             <string>Last Update</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="CopyLabel" name="nUpdate"/>
          </item>
          <item row="0" column="1">
           <widget class="CopyLabel" name="lUpdate"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Widget9">
      <attribute name="title">
       <string>&amp;Issuer</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>11</number>
       </property>
       <item>
        <widget class="DistName" name="issuer" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="unnamed_1">
      <attribute name="title">
       <string>&amp;Extensions</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>11</number>
       </property>
       <item>
        <widget class="QTextEdit" name="v3extensions">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Revocation list</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>11</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="certList">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickLabel</class>
   <extends>QLabel</extends>
   <header>widgets/clicklabel.h</header>
  </customwidget>
  <customwidget>
   <class>DistName</class>
   <extends>QWidget</extends>
   <header>widgets/distname.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CopyLabel</class>
   <extends>QLabel</extends>
   <header>widgets/clicklabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CrlDetail</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
